mojo-executor-maven-plugin/src/main/java/org/twdata/maven/mojoexecutor/plugin/MojoExecutorMojo.java

```diff
@@ -83,13 +89,47 @@
      */
     private BuildPluginManager pluginManager;
 
-    public void execute() throws MojoExecutionException {
-        if ( mavenProject.getArtifactId().equals( "mojo-executor-test-project-null-maven-project" ))
+    public void execute() throws MojoExecutionException
+    {
```

Comment:
Hi @rnc I only just noticed that your pull requests have opening braces on a new line, when most of the existing project code uses end-of-line braces.

I don't want to start a brace formatting battle, but it's probably best to maintain consistency within the project. 😄 

Unfortunately, some of the PRs that have already been merged also introduce inconsistency here.

Would you mind switching to EOL braces for now? Then if you'd like to reformat everything to use NL braces, that could be done separately. Thanks!